packages/backend/server/src/models/session.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { PrismaClient, type Session } from '@prisma/client';
+
+export type { Session };
+
+@Injectable()
+export class SessionModel {
```

Comment:
how about making multi-sessions model a all-in-one model for `session`, `userSession`

Reply:
OK, I take a try